### PR TITLE
Add charts showing past survey responses to user page

### PR DESCRIPTION
**Pull request by peter-parque**
I'll rebase this branch once #11 is merged to master, and move these charts onto the profile pages.

### CODE REVIEW

**Comment by peter-parque** [comment]
Rebased this branch onto master to get the user pages.

**Comment by peter-parque** [comment]
Rebased and now uses website from #13

File: capstone-app/src/views/LandingPage/LandingPage.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ import styles from "assets/jss/material-kit-react/views/landingPage.js";
 // Sections for this page
 import ProductSection from "./Sections/ProductSection.js";
 import SectionModal from "./Sections/SectionModal.js";
-import SectionNavbar from "./Sections/SectionNavbarLoggedin.js";
+import SectionNavbarSignup from "./Sections/SectionNavbarSignup.js";
```
Comment (dmbm0830) [inline capstone-app/src/views/LandingPage/LandingPage.js:19]: SectionNavbarSignup is not the name of the component in the file. Use SectionNavbar from SectionNavbarLoggedin instead, as that is the Navbar with the authentication built into it.

**Review by dmbm0830 — APPROVED** [review]

**Review by kidadamu — APPROVED** [review]

File: capstone-app/src/views/LandingPage/LandingPage.js (hunk 1 above)
Comment (peter-parque) [reply capstone-app/src/views/LandingPage/LandingPage.js]: Done in https://github.com/googleinterns/step160-2020/pull/12/commits/0cc4735f5c74d545d8c9b3c757a2a938d5df88a9

**Review by EmeraldBaldwin — APPROVED** [review]

File: capstone-app/src/views/ProfilePage/Sections/SurveyResponse/SurveyResponseSummary.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+      dataset.label = feeling + ' (you)';
+      dataset.backgroundColor = feelings[feeling].mainColor;
+      dataset.borderColor = feelings[feeling].mainColor;
+    } else {
+      dataset.label = feeling + ' (world)';
+      dataset.backgroundColor = feelings[feeling].bgColor;
+      dataset.borderColor = feelings[feeling].bgColor;
+    }
+
+    var intensities = {
```
Comment (sunyiping-google) [inline capstone-app/src/views/ProfilePage/Sections/SurveyResponse/SurveyResponseSummary.js:110]: I see the same var defined in another file, is there a way to reuse it?

**Review by sunyiping-google — APPROVED** [review]

File: capstone-app/src/views/ProfilePage/Sections/SurveyResponse/SurveyResponseSummary.js (hunk 2 above)
Comment (peter-parque) [reply capstone-app/src/views/ProfilePage/Sections/SurveyResponse/SurveyResponseSummary.js]: Yes! Moving it to the parent component and passing it in as a prop - 76e31e6152721fae7cca2d3980456ee17f791772.

**Pull request merged**